data/data/bootstrap/systemd/units/systemd-journal-gatewayd.service.d/certs.conf

```diff
@@ -1,4 +1,8 @@
 [Service]
+# In RHEL8 the service uses DynamicUser=yes; we need to work both ways, so hence
+# we hack this by adding the user if it doesn't exist and chown the file, rather
+# than doing it in Ignition.
+ExecStartPre=/bin/sh -c "if ! getent passwd systemd-journal-gateway &>/dev/null; then useradd -r systemd-journal-gateway; fi && chown systemd-journal-gateway: /opt/openshift/tls/journal-gatewayd.key"
```

Comment:
ques: is the user required for `key` only

Reply:
Yeah, the cert is public.